lib/spack/spack/builder.py

```diff
@@ -132,6 +132,7 @@ def __init__(self, wrapped_pkg_object, root_builder):
                     "run_tests": property(lambda x: x.wrapped_package_object.run_tests),
                     "test_log_file": property(lambda x: x.wrapped_package_object.test_log_file),
                     "test_failures": property(lambda x: x.wrapped_package_object.test_failures),
+                    "test_suite": property(lambda x: x.wrapped_package_object.test_suite),
```

Comment:
Yes. I created a `PackageTest` class in `install_test.py` to encapsulate stand-alone test-related properties and behavior.  (Would prefer a different name since it is the package test manager but I've gotten the impression naming closer to that would not fly during a review.)